docs/install/install-from-packages.md

```diff
@@ -63,43 +60,33 @@ To avoid RAM and disk swap limitations, we recommend either temporarily resizing
     If you get Plone to run on Windows alone, please feel free to document and share your process.
     ```
 
--   [Python](https://www.python.org/downloads/) 3.8, 3.9, or 3.10.
--   {term}`Cookiecutter`
+-   Python {SUPPORTED_PYTHON_VERSIONS}
+-   {term}`pipx`
 -   {term}`nvm`
--   {term}`Node.js` LTS 16.x (see "Update" note)
+-   {term}`Node.js` LTS 18.x
```

Comment:
in fact, its' 20.x (even for 17 series)

Reply:
```suggestion
-   {term}`Node.js` LTS 20.x
```

Reply:
But, wait. We recommend 18.x LTS: https://github.com/plone/volto#supported-nodejs-versions.

Reply:
We also do not mention adding support for Node.js LTS 20.x in the Volto release notes or upgrade guide, at least I could not find any mention of it.